magenta-lib/src/main/scala/magenta/artifact/S3Location.scala

```diff
@@ -96,75 +93,6 @@ object S3Artifact {
   }
 }
 
-case class S3JsonArtifact(bucket: String, key: String) extends S3Artifact {
-  val deployObjectName: String = "deploy.json"
-}
-
-object S3JsonArtifact extends Loggable {
-  def apply(build: Build, bucket: String): S3JsonArtifact = {
-    val prefix = S3Artifact.buildPrefix(build)
-    S3JsonArtifact(bucket, prefix)
-  }
-
-  def fetchInputFile(artifact: S3JsonArtifact, deprecatedPause: Option[Int])(
-    implicit client: AmazonS3, reporter: DeployReporter): Either[ArtifactResolutionError, JsonInputFile] = {
-
-    import cats.syntax.either._
-    val possibleJson = (artifact.deployObject.fetchContentAsString()(client)).orElse {
-      convertFromZipBundle(artifact, deprecatedPause)
-      artifact.deployObject.fetchContentAsString()(client)
-    }
-    possibleJson.leftMap[ArtifactResolutionError](S3ArtifactError(_)).flatMap(s =>
-      JsonInputFile.parse(s).leftMap(JsonArtifactError(_))
-    )
-  }
-
-  def convertFromZipBundle(artifact: S3JsonArtifact, deprecatedPause: Option[Int])(implicit client: AmazonS3, reporter: DeployReporter): Unit = {
-    reporter.fail(
-      """NO LONGER SUPPORTED: The artifact.zip format is no longer supported - please switch to the new format (if you
```

Comment:
good! that error was annoying because it usually meant I messed something unrelated up, but it didn't say what 🥇